tests/integration/test_podman_compose_tests.py

```diff
@@ -187,3 +187,33 @@ def test_down_with_orphans(self):
             ],
             1,
         )
+
+    def test_down_with_network(self):
+        up_cmd = [
```

Comment:
Could you put the commands to inside relevant `run_subprocess_assert_returncode`?

I do understand you copied the style from the tests, we will fix this inconsistency eventually.